test/utils/EverlongTest.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+import { VmSafe } from "forge-std/Vm.sol";
+import { BaseTest } from "test/utils/BaseTest.sol";
+import { HyperdriveTest } from "test/utils/HyperdriveTest.sol";
+import { IEverlong } from "contracts/src/interfaces/IEverlong.sol";
+import { Everlong } from "contracts/src/everlong/Everlong.sol";
+
+contract EverlongTest is HyperdriveTest {
+    IEverlong internal everlong;
+
+    function setUp() public virtual override {
+        super.setUp();
+        vm.startPrank(alice);
```

Comment:
What's the point of this? `deployEverlong` immediately switches to the `deployer`.